book3/14-objects.mkd

```diff
@@ -1,97 +1,100 @@
-so that you can zoom into 50 lines of the code and understand it
-while ignoring the other 999,950 lines of code for the moment.
+En cierto modo, la programación orientada a objetos es una forma
+de ordenar tu código de tal manera que puedas enfocarte en 50
+líneas de código y entenderlas, e ignorar las otras 999.950
+mientras tanto.
 
-Getting started
+Cómo empezar
 ---------------
 
-Like many aspects of programming, it is necessary to learn the concepts of
-object oriented programming before you can use them effectively.
-You should approach this chapter as a way to learn some terms and concepts and work
-through a few simple examples to lay a foundation for future learning.
+Al igual que muchos aspectos de la programación, es necesario aprender
+los conceptos de la programación orientada objetos para poder utilizarlos
+de manera efectiva. Deberías enfocarte en este capítulo como una forma
+de aprender algunos términos y conceptos y examinar algunos ejemplos
+sencillos para sentar las bases de tu futuro aprendizaje.
 
-The key outcome of this chapter is to have a basic understanding of how
-objects are constructed and how they function and most importantly how
-we make use of the capabilities of objects that are provided to us by Python
-and Python libraries.
+El resultado clave de este capítulo será una comprensión a nivel básico
+de cómo se construyen los objetos, cómo funcionan y, lo más importante,
+cómo usar las características de los objetos que nos dan Python
+y sus librerías.
 
-Using objects
+Usando objetos
 -------------
 
-As it turns out, we have been using objects all along in this book. Python
-provides us with many built-in objects. Here is some simple code
-where the first few lines should feel very simple and natural to you.
+Curiosamente, en el libro hemos estado utilizando objetos todo este tiempo. Python
+contiene muchos objetos incluidos. He aquí un programa sencillo, cuyas
+primeras líneas deberían resultarte sumamente simples y familiares.
 
 \index{list object}
 
 \VerbatimInput{../code3/party1.py}
 
-Instead of focusing on what these lines accomplish, let's look at what is
-really happening from the point of view of object-oriented programming.
-Don't worry if the following paragraphs don't make any sense the
-first time you read them because we have not yet defined all of these terms.
+En lugar de enfocarnos en el resultado que obtienen estas líneas, enfoquémonos
+en lo que está pasando desde el punto de vista de la programación orientada
+a objetos. No te preocupes si los siguientes párrafos no parecen tener sentido
+la primera vez que los lees, pues no hemos definido todos estos términos aún.
 
-The first line *constructs* an object of type `list`, the second and third
-lines *call* the `append()` *method*, the fourth line calls
-the `sort()` method, and the fifth line *retrieves* the item at position 0.
+La primera línea *construye* un objeto de tipo 'list', la segunda y tercera
```

Comment:
In this specific case, use backticks instead of single quotes, as it will render as a code-block. Same for the "sort()" word in line 72, "append()" on line 71 (add the parenthesis), and __getitem__() in line 74.